Jenkinsfile

```diff
@@ -158,6 +159,40 @@ pipeline {
 	     */
             //failFast true
             parallel {
+                stage('Build RPM on CentOS 7') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile-mockbuild.centos:7'
+                            dir 'utils/docker'
+                            label 'docker_runner'
+                            additionalBuildArgs  '--build-arg UID=$(id -u)'
+                            args  '--group-add mock --cap-add=SYS_ADMIN --privileged=true'
+                        }
+                    }
+                    steps {
+                        sh '''rm -rf artifacts/
```

Comment:
Didn't we decide the best way to delete prior builds was to use the following in post:

                        cleanup {
                            dir('artifacts') {
                                deleteDir()
                            }
                        }

Reply:
I don't recall that conversation but that could just be my 50 year-old memory.

But generally speaking I usually like to try to preserve state at the end and clean it up at the start (i.e. of the next run) for the cases where it's useful to inspect the state at the end in the case of failures.

That's not a hard rule for me, so I could be talked into changing to clean up at the end if you really prefer it.

Reply:
As always my preference is for consistency, that we decide best practice and apply it.  That's what you had me do for the IOF build step.